spaces/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Address(models.Model):
+    street_name = models.CharField(max_length=140)
+    civic = models.CharField(max_length=20)
```

Comment:
I think I didn't explain it very well. That's what I was referring to: https://en.wikipedia.org/wiki/House_numbering